.github/workflows/build.yml

```diff
@@ -886,7 +886,13 @@ jobs:
       - uses: pnpm/action-setup@v4
         with:
           version: 9.4.0
+     - name: Set up Node for Angular 20
```

Comment:
This is not ideal because we should use the same Node version to build the SDK and to build && run the app. But since we're blocked on upgrading to 20.19.0, I think this is a fair compromise for now.